.eslintrc.json

```diff
@@ -1,28 +1,29 @@
 {
-  "parserOptions": {
-    "ecmaVersion": 5
-  },
+  "env": { "commonjs": true },
   "extends": "eslint:recommended",
-  "env": {
-    "commonjs": true
-  },
+  "parserOptions": { "ecmaVersion": 5 },
   "rules": {
-    "strict": [2, "global"],
-    "block-scoped-var": 2,
-    "consistent-return": 2,
-    "eqeqeq": [2, "smart"],
-    "guard-for-in": 2,
-    "no-caller": 2,
-    "no-extend-native": 2,
-    "no-loop-func": 2,
-    "no-new": 2,
-    "no-param-reassign": 2,
-    "no-return-assign": 2,
-    "no-unused-expressions": 2,
-    "no-use-before-define": 2,
-    "radix": [2, "always"],
-    "indent": [2, 2],
-    "quotes": [2, "double"],
-    "semi": [2, "always"]
+    "block-scoped-var": "error",
+    "consistent-return": "error",
+    "eqeqeq": "error",
+    "guard-for-in": "error",
+    "no-bitwise": "error",
+    "no-caller": "error",
+    "no-extra-parens": "off",
+    "no-extend-native": "error",
+    "no-loop-func": "error",
+    "no-new": "error",
+    "no-param-reassign": "error",
+    "no-return-assign": "error",
+    "no-sequences": "error",
+    "no-unused-expressions": "error",
+    "no-use-before-define": "error",
+    "no-undef": "error",
+    "no-eq-null": "error",
+    "radix": ["error", "always"],
+    "indent": ["error", 2, { "SwitchCase": 1 }],
```

Comment:
Original file had `[2, 2]`. Why was this change made?

Reply:
This rule was based on the same rule used in `Aff` and it's just changed here for consistency.

Reply:
Gotcha. Then let's merge this in.